### PR TITLE
Add Korean localization for Dashboard

### PR DESCRIPTION
**Pull request by seokho-son**
This PR is to initiate Korean localization for Dashboard.

This PR modifies (1) i18n/locale_conf.json, (2) package.json to add "ko" as a new localization language.
Also, the PR includes **i18n/messages.ko.xlf** that includes actual translation contents.

I am one of @kubernetes/sig-docs-ko-owners , and we are happy to know about new Kubernetes Dashboard v2.0.0 and provide localization for Korean !! (Thank you @shu-mutou for your guidance) 
 
cc to sig-docs-ko-reviews for checking any major issues in the Korean translation. 
(Please note that this PR is for initiation of Korean localization for Dashboard. I did my best for the translation, but if there is a major issue, please let me know :) )
/cc @kubernetes/sig-docs-ko-reviews 

image

### CODE REVIEW

**Comment by shu-mutou** [comment]
Thank you for adding new l10n file! But a typo was found yesterday, and we need to refresh messages files now. Please wait #4173 merged. Then rebase to it, run `npm run fix:i18n` again and recheck your i18n file. Sorry for inconvinience.

**Comment by seokho-son** [comment]
> Thank you for adding new l10n file! But a typo was found yesterday, and we need to refresh messages files now. Please wait #4173 merged. Then rebase to it, run `npm run fix:i18n` again and recheck your i18n file. Sorry for inconvinience.

I got it. I will rebase this PR after #4173 is merged. Thank you.
In the meantime, I hope to get comments from reviewers :)
/hold

**Comment by shu-mutou** [comment]
#4173 was merged. Please rebase this PR. Thanks. 😉

File: i18n/messages.ko.xlf [hunk 1]
```diff
@@ -0,0 +1,5121 @@
...
+          <context context-type="linenumber">164</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">../src/app/frontend/resource/workloads/deployment/detail/template.html</context>
+          <context context-type="linenumber">254</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="4a4f46a2dcec36bd5c8c371ceee55c2226dec27f" datatype="html">
+        <source>Schedule</source>
+        <
```
Comment (shu-mutou) [inline i18n/messages.ko.xlf]: This typo ("requestes") was fixed few days ago, so  "trans-unit id" was changed. Please re-run `npm run fix:i18n` after rebasing, then check again this files. After running `npm run fix:i18n`, this typo will fixed and some "transunit" for new views will be added.

**Comment by seokho-son** [comment]
@shu-mutou Thank you. I applied the fixed typo and translated the associated target into Korean.

**Comment by shu-mutou** [comment]
/lgtm
@seokho-son Great!! Thank you for addressing updates. If you have inconvinience about our i18n workflow or have ideas to improve, feel free to submit issue.

**Comment by floreks** [comment]
Great work. Thanks!

/lgtm

**Pull request merged**